addons/mail/static/src/components/rtc_video/rtc_video.js

```diff
@@ -60,11 +60,21 @@ export class RtcVideo extends Component {
     //--------------------------------------------------------------------------
 
     /**
+     * Plays the video as some browsers may not support or block autoplay.
+     *
      * @private
      * @param {Event} ev
      */
     async _onVideoLoadedMetaData(ev) {
-        await ev.target.play();
+        try {
+            await ev.target.play()
+        } catch (error)  {
+            if (error.name != 'NotAllowedError') {
```

Comment:
`!==`, also check that the type of error is object before trying to access a props on it